docs/learn.md

```diff
@@ -126,10 +126,281 @@ OCaml. Some examples are the upgrade of the compiler version used by Melange to
 4.12, or modelling the changes to the OCaml compiler that Melange uses as just a
 plain library, instead of a full fork of the whole compiler.
 
-## Build system and package management
+## Package management
+
+Melange uses a dual approach to package management:
```

Comment:
this is outright confusing. How about we say:

> Melange can consume packages from both NPM and the OPAM repository. Through Dune, it also provides great workspace vendoring support.

Reply:
I left the Dune part out as I thought it would be meaningless for someone new to Dune and opam ("workspace vendoring" is pretty Dune specific nomenclature).